PowervRA/Functions/Public/config-service/Set-vRAHostname.psm1

```diff
@@ -0,0 +1,83 @@
+configure hostname
+
+get the node id of the appliance
+
+name is mandatory
+
+function Set-vRAHostname {
+<#
+    .SYNOPSIS
+    Set the hostname value used to resolve the application.
+    
+    .DESCRIPTION
+    This does not change the appliance hostname, only the hostname alias (typically the load balancer VIP FQDN.)
+
+    .PARAMETER Name
+    The name of the node on which to execute the command.
+
+    .PARAMETER Hostname
+    The desired hostname value.
+    
+    .INPUTS
+    System.String
+
+    .OUTPUTS
+    System.Management.Automation.PSObject.
+
+    .EXAMPLE 
+    Set-vRAHostname -Name vra-01a.lab.local -Hostname vra.lab.local
+
+#>
+[CmdletBinding(DefaultParameterSetName="Standard")][OutputType('System.Management.Automation.PSObject')]
```

Comment:
You don't need to specify a default parameter set name here as all of the params are in all parameter sets.

Also this is a Set function so we need to support ShouldProcess & ConfirmImpact

Change line 31 to:

```
[CmdletBinding(SupportsShouldProcess,ConfirmImpact="High")][OutputType('System.Management.Automation.PSObject')]
```